Specifications/Protobuf/for_Extensions/when_converting_execution_context_to_protobuf.cs

```diff
@@ -21,7 +21,7 @@ public class when_converting_execution_context_to_protobuf
             execution_context = new ExecutionContext(
                 Guid.NewGuid(),
                 Guid.NewGuid(),
-                Guid.NewGuid(),
+                Versioning.Version.NotSet,
```

Comment:
Perhaps it's better to test with an actual Version? Just worried that `NotSet` is a default value somewhere so that we won't see if it is actually passed through correctly.

Reply:
Agree